packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -64,6 +68,12 @@ export const CustomScrollbar = ({
     }
   }, [scrollTop]);
 
+  useEffect(() => {
+    if (ref.current && scrollToRowIndex != null && height) {
+      ref.current.scrollTop(scrollToRowIndex * height);
```

Comment:
Why not use scrollTop prop? this height* scrollToRowIndex seems like something that should not be inside CustomScrollbar

Reply:
good catch, looks like we can simply set the initial state of `rowHighlightIndex` in the RowsList and all of this functionality is already built